portfolio/src/main/webapp/scriptTutorial2.js

```diff
@@ -1,8 +1,27 @@
-/** Fetches the hardcoded message from the server and adds it to the page. */
-async function showHardCodedString() {
-  const responseFromServer = await fetch('/hello');
-  const textFromResponse = await responseFromServer.text();
+/** Fetches the JSON string from the server and adds it to the page. */
+async function showRandomMessage() {
+  const responseFromServer = await fetch('/randomMessage');
+  const randomMessage = await responseFromServer.json();
+
+  //console.log(randomMessage);
 
   const stringContainer = document.getElementById('string-container');
-  stringContainer.innerText = textFromResponse;
+  const message = randomMessage[Math.floor(Math.random() * randomMessage.length)];
+  stringContainer.innerText = message;
+
+}
+
+/**
+ * Adds a random fun fact to the page.
+ */
+function addRandomFunFact() {
```

Comment:
showRandomMessage & addRandomFunFact seem to be a little confusingly named. Its better to follow a naming convention & especially that is more descriptive of the function.

For example you could do something like:
showHarryPotterQuote
showRandomFunFact

or something like that..

Reply:
thank you !